eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -243,20 +243,21 @@ extends:
                 WorkingDirectory: $(SdkRepoDirectory)
                 ScriptDirectory: $(SdkRepoDirectory)/eng/common/scripts
 
-            - task: PowerShell@2
-              displayName: Create pull request
-              condition: and(succeeded(), eq(variables['HasChanges'], 'true'), ne(variables['Build.Reason'], 'PullRequest'))
-              inputs:
-                pwsh: true
-                workingDirectory: $(SdkRepoDirectory)
-                filePath: $(SdkRepoDirectory)/eng/common/scripts/Submit-PullRequest.ps1
-                arguments: >
-                  -RepoOwner "$(SdkRepoOwner)"
-                  -RepoName "$(SdkRepoName)"
-                  -BaseBranch "main"
-                  -PROwner "azure-sdk"
-                  -PRBranch "$(PrBranch)-$(Build.BuildId)"
-                  -AuthToken "$(azuresdk-github-pat)"
-                  -PRTitle "$(PrTitle)-generated-from-$(Build.DefinitionName)-$(Build.BuildId)"
-                  -PRBody "$(GeneratedSDKInformation)"
-                  -OpenAsDraft $true
+            - ${{ if not(endsWith(variables['SdkRepoName'], '-pr')) }}:
```

Comment:
@raych1 can we disable this for any pull request validation run? Only have the PR created for manual runs?

Reply:
i think that is already being handled by the step condition.

Reply:
Yes, pull request is already being excluded from SDK PR creation.

Reply:
Then why do we need to special case the private repo here?

Reply:
The reason is that we don’t want to create a PR for the release scenario either when the spec repository is private.